src/Pages/Main/Component/Carousel/Carousel.scss

```
@@ -0,0 +1,70 @@
+@import 'src/Styles/_variables.scss';
+.carouselContainer {
```

```suggestion
@import 'src/Styles/_variables.scss';

.carouselContainer {
```
구분되는 요소들끼리는 한줄 띄워주세요